duckduckgo/duckduckgo-privacy-extension
# Tracker Networks subview - split from grade details

**Reviewer:** @laurengarcia 



## Description:
Implement the tracker networks subview as shown in Zeplin, and link to it from the main popup.


<img width="303" alt="screen shot 2018-01-10 at 12 38 15 pm" src="https://user-images.githubusercontent.com/3652195/34786812-2dd91d8a-f603-11e7-9989-b628f8b654d7.png">

<img width="303" alt="screen shot 2018-01-10 at 12 56 52 pm" src="https://user-images.githubusercontent.com/3652195/34787591-c6958c82-f605-11e7-8a6f-828a83e53d02.png">


<img width="303" alt="screen shot 2018-01-10 at 6 17 13 pm" src="https://user-images.githubusercontent.com/3652195/34800626-9d3aabe2-f632-11e7-9f32-5e6f8c40054b.png">

<img width="301" alt="screen shot 2018-01-10 at 6 17 04 pm" src="https://user-images.githubusercontent.com/3652195/34800627-9d4837da-f632-11e7-9a5f-3c322c50d3ac.png">

## Steps to test this PR:

1. Build extension and reload
2. The tracker networks line in the main popup should look like in Zeplin (and have the correct number of trackers)
3. Clicking on the tracker networks line should open the tracker networks subview
4. The subview should look like in the mockup and have the correct information about trackers
5. Clicking on the left arrow icon in the header should slide back to the main popup view


## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
